spec/controllers/builds_controller_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'spec_helper'
 
 describe BuildsController do
+  render_views
```

Comment:
this can be placed inside of the "when repository is disabled" context. That way it isn't enabled for the whole file. The reason is `render_views` slows down the tests a little bit.